backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/endpoint/EndpointResource.java

```diff
@@ -208,36 +201,15 @@ public List<NotificationHistoryDTO> getEndpointHistory(@Context SecurityContext
                 schema = @Schema(type = SchemaType.INTEGER)
             )
     })
+    @Authorization(legacyRBACRole = RBAC_READ_INTEGRATIONS_ENDPOINTS, workspacePermissions = INTEGRATIONS_VIEW)
```

Comment:
This is a different kind of permission replacement. Before, we filtered on authorized integrations (below in this diff). Now, we just check that the user is allow to view all integrations.